.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: php
 
 php:
-  - 5.5
+  - 5.5.9
```

Comment:
Can't we drop 5.* completely? Event 2.8 runs with 7.x so no need to keep the 5 in?

Reply:
we should use the same version as symfony 3.0 requires.

Reply:
Restored